web-app/sections/technology/sections/timeline/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import TimelineData from 'modules/timeline-data'
+import s from './index.scss'
+
+Timeline.propTypes = {
+  phases: PropTypes.array.isRequired
+}
+
+function Timeline({ phases, ...rest }) {
+  // console.log(phases)
```

Comment:
Not sure if you want to leave this comment in or if you use it for debugging, but if its pretty stable seems safe to delete.